tests/views/test_sample_parameters_buttons.py

```diff
@@ -525,8 +526,53 @@ def test_help_button(qtbot):
     dialog.close()
 
 
+@pytest.mark.skip(reason="This test is not working on Ubuntu_22.04.")
```

Comment:
Can you add a task to fix this test.

Reply:
This defect is tracked with [defect1386](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=1386)